qemu/tests/block_disk_not_default_options.py

```diff
@@ -0,0 +1,30 @@
+from virttest import error_context
+
+from provider.block_devices_plug import BlockDevicesPlug
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test the disk non-default options.
+
+    Steps:
+        1. Boot a VM with packed=true and page-per-vq=true
+        2. Do some basic I/O operation in the VM
+        3. Unplug and hotplug the disk
+        4. Do again a basic I/O operation
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=360)
+    session.cmd(params.get("dd_cmd"))
+
```

Comment:
if possible, consider adding qtree verification for the option

Reply:
Verification included !